apps/src/sites/studio/pages/teacher_dashboard/show.js

```diff
@@ -55,6 +59,10 @@ $(document).ready(function () {
     currentUser,
     sectionStandardsProgress,
     locales,
+    viewAs,
```

Comment:
Needed to add some reducers that the course page relies on and were not already in teacher_dashboard